src/dnx.windows/dnx.exe.manifest

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
```

Comment:
I believe so, but I'm not sure. Also those are Macros, so they'd have to be added in the native code. I think that would be a viable option once we get closer to having native code pass data to managed code.